docs/install/install-jetson.md

```diff
@@ -60,49 +59,54 @@ sudo pip install \
 ```
 
 If you plan to cross-compile you will need to install these dependencies on that computer as well.
+If you get an error about something being busy, you can restart the nano and this error will go away and you can continue installation of the prerequisites.
 
-### Configure CUDA
+### Download the source & setup some environment variables:
 
-You can check to see what version of CUDA is running with `nvcc`.
+These steps are optional, but some of the following instructions expect MXNet source files and the `MXNET_HOME` environment variable. Also CUDA commands will not work out of the box without updating your path.
+
+Clone the MXNet source code repository using the following `git` command in your home directory:
 
 ```bash
-nvcc --version
+git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
 ```
 
-To switch CUDA versions on a device or computer that has more than one version installed, use the following and replace the version as appropriate.
+Setup your environment variables for MXNet and CUDA in your `.profile` file in your home directory.
+Add the following to the file.
 
 ```bash
-sudo rm /usr/local/cuda
-sudo ln -s /usr/local/cuda-10.0 /usr/local/cuda
+export PATH=/usr/local/cuda/bin:$PATH
+export MXNET_HOME=$HOME/mxnet/
+export PYTHONPATH=$MXNET_HOME/python:$PYTHONPATH
 ```
 
-**Note:** When cross-compiling, change the CUDA version on the host computer you're using to match the version you're running on your Jetson device.
-**Note:** CUDA 10.1 is recommended but doesn't ship with the Nano's SD card image. You may want to go through CUDA upgrade steps first.
+You can then apply this change immediately with the following:
+```bash
+source .profile
+```
 
-### Download the source & setup some environment variables:
+**Note:** Change the `~/.profile` steps according to how you prefer to use your shell. Otherwise, your environment variables will be gone after you logout.
 
-These steps are optional, but some of the following instructions expect MXNet source files and the `MXNET_HOME` environment variable.
+### Configure CUDA
 
-Clone the MXNet source code repository using the following `git` command in your home directory:
+You can check to see what version of CUDA is running with `nvcc`.
 
 ```bash
-git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet
-cd mxnet
+nvcc --version
 ```
 
-Setup your environment variables for MXNet.
+To switch CUDA versions on a device or computer that has more than one version installed, use the following and replace the version as appropriate.
 
 ```bash
-cd ..
-export MXNET_HOME=$(pwd)
-echo "export PYTHONPATH=$MXNET_HOME/python:$PYTHONPATH" >> ~/.rc
-source ~/.rc
+sudo rm /usr/local/cuda
```

Comment:
Maybe add a line here that describes how these commands will remove the directory and then replace it by creating a symlink. But Linux users may not need this called out. :)

Reply:
It was always a symlink, not an actual folder. I just briefly freaked out when I thought I deleted the cuda files.